tests/frequentist/test_ztest.py

```diff
@@ -202,8 +202,8 @@ def test_powered_effect(self):
         assert np.isclose(powered_effect[POWERED_EFFECT][1], 0.5291, atol=0.001)
         assert np.isclose(powered_effect[POWERED_EFFECT][2], 0.4596, atol=0.001)
         assert np.isclose(powered_effect[POWERED_EFFECT][3], 0.4869, atol=0.001)
-        assert powered_effect[REQUIRED_SAMPLE_SIZE][0] == float("inf")
```

Comment:
When NIMs are None, we now return sample size None rather than inf, which makes more sense.